### PR TITLE
Add player preference

### PR DESCRIPTION
**Pull request by Indhumathi040398**
1. Added player preference widget
2. Added string constants
3. Added image constants

### CODE REVIEW

File: lib/constants/image_constants.dart [hunk 1]
```diff
@@ -1,3 +1,4 @@
 class ImageConstants {
   static const badmintonPlayer = "assets/images/badminton_player.png";
+  static const iconPlayer = "assets/images/player_Vector.png";
```
Comment (kingmanimarankv) [inline lib/constants/image_constants.dart:3]: Rename the file name and follow convention. File names should be snake case lower case

File: lib/constants/string_constants.dart [hunk 2]
```diff
@@ -3,4 +3,14 @@ class StringConstants {
...
   static const bookNow = "Book Now";
+  static const playerPreference ="Here is why more players prefer us?";
+  static const playerRow1 ="Pro Coaching";
+  static const playerRow2 = "All Ages";
+  static const playerRow3 = "Memberships";
+  static const playerRow4 ="All Skill Levels";
+  static const playerRow11 ="Our mission is to ensure every individual in Ontario has the access to quality badminton coaching";
+  static const playerRow22 ="We have suitable training programs from kids to adult. Enroll now to fast-track your badminton coaching";
+  static const playerRow33 ="We offer different membership programs to enhance the hassle free experinence in playing.";
+  static const playerRow44 = "We have suitable training programs from begginner level to Professional level to excel in the game";
```
Comment (kingmanimarankv) [inline lib/constants/string_constants.dart:14]: Rename the variables with proper meaningful names

File: lib/screens/widgets/player_perference.dart [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+import '../../constants/image_constants.dart';
+import '../../constants/string_constants.dart';
+
+class PlayerPreference extends StatelessWidget {
```
Comment (kingmanimarankv) [inline lib/screens/widgets/player_perference.dart:7]: Rename the class to PreferencesSection

File: lib/screens/widgets/player_perference.dart [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  PlayerPreference({super.key});
+  final Map<String,String> content ={
+    StringConstants.playerRow1:StringConstants.playerRow11,
+    StringConstants.playerRow2:StringConstants.playerRow22,
+    StringConstants.playerRow3:StringConstants.playerRow33,
+    StringConstants.playerRow4:StringConstants.playerRow44,
```
Comment (kingmanimarankv) [inline lib/screens/widgets/player_perference.dart:13]: Replace the StringConstant variable names

File: lib/screens/widgets/player_perference.dart [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                  ),
+                ),
+            ) ).toList()
+            
+            
+            
+              
+             
+             
+
```
Comment (kingmanimarankv) [inline lib/screens/widgets/player_perference.dart:57]: Format the code before commiting the changes

**Review by kingmanimarankv — CHANGES_REQUESTED** [review]

File: lib/constants/image_constants.dart (hunk 1 above)
Comment (Indhumathi040398) [reply lib/constants/image_constants.dart:3]: Made the changes as mentioned

File: lib/constants/string_constants.dart (hunk 2 above)
Comment (Indhumathi040398) [reply lib/constants/string_constants.dart:14]: Updated the variable names

File: lib/screens/widgets/player_perference.dart (hunk 3 above)
Comment (Indhumathi040398) [reply lib/screens/widgets/player_perference.dart:7]: Renamed the class name

File: lib/screens/widgets/player_perference.dart (hunk 4 above)
Comment (Indhumathi040398) [reply lib/screens/widgets/player_perference.dart:13]: Updated the changes mentioned

File: lib/screens/widgets/player_perference.dart (hunk 5 above)
Comment (Indhumathi040398) [reply lib/screens/widgets/player_perference.dart:57]: Ok

**Review by kingmanimarankv — APPROVED** [review]

**Pull request merged**